server/api/hook.go

```diff
@@ -269,7 +269,7 @@ func PostHook(c *gin.Context) {
 }
 
 // TODO: parse yaml once and not for each filter function
-func branchFiltered(build *model.Build, remoteYamlConfigs []*remote.FileMeta) (bool, error) {
+func checkIfFiltered(build *model.Build, remoteYamlConfigs []*remote.FileMeta) (bool, error) {
```

Comment:
```suggestion
// Check if at least one pipeline config will be execute otherwise we will just ignore this webhook
func checkIfFiltered(build *model.Build, remoteYamlConfigs []*remote.FileMeta) (bool, error) {
```